examples/with-actix/README.md

```diff
@@ -0,0 +1,25 @@
+# Actix Web Example
+
+This is an example of using [Prisma Client Rust](https://github.com/Brendonovich/prisma-client-rust) with [Actix Web](https://actix.rs/) and connect with [Prisma Data Platform](https://www.prisma.io/data-platform).
```

Comment:
Where is the Data Platform being used?

Reply:
I'm use MySQL because it can connect to database with connection string that will make this example have env file example for who don't know how to use with env. And can connect to Prisma Data Platform with MySQL connection string that generated from Prisma Data Platform

Reply:
While that is understandable, I'd rather use SQLite for majority of the examples, and then have a dedicated example that includes using `dotenv` and Prisma Data Platform. The Actix example should focus solely on Actix integration.

Reply:
Okay I'm changed to SQLite.